src/main/java/org/terasology/scenario/internal/ui/LogicTree.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.scenario.internal.ui;
+
+import org.terasology.rendering.nui.widgets.treeView.Tree;
+
+public class LogicTree extends Tree<LogicTreeValue> {

```

Comment:
Yeah, possibly. Especially with the change to the logicTree being entirely the visual portion basically and the main structure being built from entities. Will have to see how it effects future progress.